README.md

```diff
@@ -13,7 +13,7 @@ Please see the alternate branches [gnome-3.0](https://github.com/paradoxxxzero/g
 This is the recommended install method and offers One Click Install via [extensions.gnome.org](https://extensions.gnome.org/extension/120/system-monitor/)
 
 #### Dependencies:
-    
+
```

Comment:
Whoops, my editor is set to kill all excess white space on save (`autocmd BufWritePre * :%s/\s\+$//e`) -- this isn't directly related to this commit, so I can back this and the one below out if you're feelin' picky, no probs, just lemme know.